### PR TITLE
qemu: Patched CVE-2021-3929, CVE-2021-4207

### PR DESCRIPTION
**Pull request by liunan-ms**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add patches for Qemu to fix CVE-2021-3929, CVE-2021-4207
###### Change Log  

- Patched CVE-2021-3929, CVE-2021-4207

###### Does this affect the toolchain?  

**NO**

###### Links to CVEs  
- https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2021-3929
- https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2021-4207

###### Test Methodology

- Pipeline build id: Buddy build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=280371&view=results

### CODE REVIEW

**Review by mfrw — APPROVED** [review]
LGTM

File: SPECS/qemu/qemu.spec [hunk 1]
```diff
@@ -2294,6 +2298,9 @@ useradd -r -u 107 -g qemu -G kvm -d / -s %{_sbindir}/nologin \
 
 
 %changelog
+* Tue Dec 20 2022 Nan Liu <liunan@microsoft.com> - 6.2.0-13
+- Adress CVE-2021-3929, CVE-2021-4207
```
Comment (amritakohli) [inline SPECS/qemu/qemu.spec:2302]: Address* (spelling)

File: SPECS/qemu/qemu.spec [hunk 2]
```diff
@@ -2294,6 +2298,9 @@ useradd -r -u 107 -g qemu -G kvm -d / -s %{_sbindir}/nologin \
...
+
 * Mon Dec 19 2022 Nan Liu <liunan@microsoft.com> - 6.2.0-12
 - Adress CVE-2022-3872
```
Comment (amritakohli) [inline SPECS/qemu/qemu.spec:2305]: Address

File: SPECS/qemu/qemu.spec (hunk 1 above)
Comment (liunan-ms) [reply SPECS/qemu/qemu.spec]: Ah-Ha, I didn't notice that. Thanks for pointing it out. Fixed all the spelling issues.

**Pull request merged**